shell/platform/common/cpp/client_wrapper/include/flutter/event_stream_handler.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_CPP_CLIENT_WRAPPER_INCLUDE_FLUTTER_EVENT_STREAM_HANDLER_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_CPP_CLIENT_WRAPPER_INCLUDE_FLUTTER_EVENT_STREAM_HANDLER_H_
+
+#include "event_sink.h"
+
+namespace flutter {
+
+// Handler of stream setup and tear-down requests.
+// Implementations must be prepared to accept sequences of alternating calls to
+// onListen(T, EventSink) and onCancel(Object). Implementations should ideally
+// consume no resources when the last such call is not onListen(). In typical
+// situations, this means that the implementation should register itself with
+// platform-specific event sources onListen() and deregister again onCancel().
+template <typename T>
+class StreamHandler {
+ public:
+  // Handles a request to set up an event stream.
+  // @param arguments stream configuration arguments, possibly null.
```

Comment:
Why is this notation being used only for a subset of this one class?

I'm fine with adopting Doxygen for the client wrapper, but if it's going to be done it should be done consistently in all the new code.

Reply:
I used the comments of following source code for reference. I thought the same comment would be better.
`shell/platform/android/io/flutter/plugin/common/EventChannel.java`

There's definitely no sense of unity in the code. Adjust to other C++ code, I try not to use Doxygen Doc comments.